src/org/parosproxy/paros/extension/state/ExtensionState.java

```diff
@@ -74,39 +71,10 @@ public void hook(ExtensionHook extensionHook) {
 	    // ZAP: Changed to check if there is a view.
 	    if (getView() != null) {
 		    extensionHook.getHookMenu().addEditMenuItem(extensionHook.getHookMenu().getMenuSeparator());
-		    extensionHook.getHookMenu().addEditMenuItem(getMenuSessionTrackingEnable());
 		    extensionHook.getHookMenu().addEditMenuItem(getMenuResetSessionState());
 	    }
 	}
 
-	
-	/**
-	 * This method initializes menuViewImage	
-	 * 	
-	 * @return javax.swing.JCheckBoxMenuItem	
-	 */    
-	private JCheckBoxMenuItem getMenuSessionTrackingEnable() {
-		if (menuSessionTrackingEnable == null) {
-		    menuSessionTrackingEnable = new JCheckBoxMenuItem();
-		    menuSessionTrackingEnable.setText(Constant.messages.getString("menu.edit.enableTracking"));	// ZAP: i18n
-			getMenuResetSessionState().setEnabled(menuSessionTrackingEnable.isSelected());
-
-			menuSessionTrackingEnable.addItemListener(new java.awt.event.ItemListener() { 
-
-				@Override
-				public void itemStateChanged(java.awt.event.ItemEvent e) {    
-
-					getModel().getOptionsParam().getConnectionParam().setHttpStateEnabled(menuSessionTrackingEnable.isEnabled());
-					getMenuResetSessionState().setEnabled(menuSessionTrackingEnable.isSelected());

```

Comment:
We need more changes, the reset menu item will stop working as is (it's not being enabled/disabled based on the state of the new option).